### ISSUE TITLE
Permissions system for new user groups does not work correctly.

### ISSUE THREAD

**Issue by olairmarcos** [intelliants/subrion#260]
I posted this problem in the forum, I had no answer. I will post here again.
Permissions system for new user groups does not work correctly.
How to test:
1- Create a new user group. Copy administrator permissions.
![2016-12-29_231701](https://cloud.githubusercontent.com/assets/8938718/21558378/150e4db8-ce20-11e6-9b2d-5f49910c11d4.jpg)
-New user group created.
![2016-12-29_231800](https://cloud.githubusercontent.com/assets/8938718/21558382/1e6e527c-ce20-11e6-8dcc-6d02039b96eb.jpg)
-Manually configure the permissions (frontpage). 
![2016-12-29_232114](https://cloud.githubusercontent.com/assets/8938718/21558411/985475a8-ce20-11e6-927c-998819e1bfa9.jpg)
-Manually configure the permissions (panel control). 
![2016-12-29_232835](https://cloud.githubusercontent.com/assets/8938718/21558421/ce5fb612-ce20-11e6-9259-103f19e1fe20.jpg)

2- Add a new user to the new group.
![2016-12-29_233329](https://cloud.githubusercontent.com/assets/8938718/21558424/dcb80b38-ce20-11e6-8943-266f61270980.jpg)
-New user created.
![2016-12-29_233356](https://cloud.githubusercontent.com/assets/8938718/21558426/e30f4f1e-ce20-11e6-8286-368ed6494b24.jpg)
-See configuration the permissions (frontpage).
![2016-12-29_233441](https://cloud.githubusercontent.com/assets/8938718/21558445/48fa5170-ce21-11e6-821f-f0550eb8e973.jpg)
-See configuration the permissions (panel control).
![2016-12-29_233534](https://cloud.githubusercontent.com/assets/8938718/21558446/4f0db462-ce21-11e6-81bd-59016f41cb6e.jpg)

3- Test the permissions of the new user in the control panel.
 Content > Pages = Working
Dashboard > Uploads = Working
All other settings have a 403 error, when we try to configure it on the control panel.
I see that the permissions user for the new user groups do not work correctly.

Thank you for listening.

**Comment by Batry** [comment]
Thanks for such a detailed report @olairmarcos. We will check it.

**Comment by Batry** [comment]
Partially duplicates and references to #237 which has been solved.

**Comment by Batry** [comment]
Unable to reproduce, everything works fine.

**Comment by vbezruchkin** [comment]
@Batry the problem exists. Follow the instructions as described.

1. Create usergroup Managers based on admin permissions
2. Create a new user of Managers group
3. Try to login to admin dashboard

I am unable to login as admin. Check permissions after usergroup creation - they are not similar to administrator permissions. Recheck carefully, @Batry

**Comment by vbezruchkin** [comment]
Yep, default permissions for usergroups are incorrect. Here is the way to go:

1. There should be only one administrators usergroup - prevent permissions creation on top of this group. Delete it from the dropdown on create usergroup page.
2. Recheck default permissions for moderators group - they should have access admin dashboard

Thanks
